pom.xml

```diff
@@ -364,6 +364,13 @@ under the License.
             </excludes>
           </configuration>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <configuration>
+            <configLocation>config/maven_checks_nocodestyle.xml</configLocation>
```

Comment:
I don't think this is needed.  Just of the upgrade of the parent worked for me.

Reply:
build is ok, but site report of checkstyle will contains all old issues